packages/celotool/package.json

```diff
@@ -7,6 +7,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@celo/verification-pool-api": "^1.0.0",
+    "@celo/celocli": "~0.0.30-beta2",
```

Comment:
Do you intend to use master version of a fix (already deployed) version of cli...

You're using a fix version here (as you can see for the changes in yarn.lock

Reply:
If there is a new version of the cli i can update to it, right now `0.0.30-beta2` is latest. Also it should update to the next minor version in case there is a fix (because of `~`).

Reply:
My point here is that you can choose between two poisons :P

1) Don't use a fixed cli version, instead use whatever is on the cli package on that same commit. So you don't have to worry about versioning, outdated packages, releases each time you change something. But, you need to make sure that when building celotool dockerfile cli is included.

2) Use a fixed package for cli. Which at some point will be out of date. But you know that no matter where your run `npm install` there's a version out there for you.

I think probably (1) is the best choice.

Reply:
ok, will go for (1)